scania-digital-design-system/sdds
# feat(table): sorting in data-table as web component

**Describe pull-request**  
Make data table with sorting functionality

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#1787](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1787), [AB#1696](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1696), [AB#1780](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1780)

**How to test**  
_Add a description of how to test if possible_
1. Go to https://task-1696.d1g8v8mrkx992n.amplifyapp.com/?path=/story/component-table--web-component
2. Check Web Component under Table


**Additional context**  
Figma prototype: https://www.figma.com/proto/ZUT3Te7JVWggoukL9QGsPp/Components?page-id=4%3A6&node-id=948%3A99358&viewport=241%2C48%2C0.53&scaling=min-zoom&starting-point-node-id=948%3A99358&show-proto-sidebar=1

Figma design: https://www.figma.com/file/ZUT3Te7JVWggoukL9QGsPp/Components?node-id=4%3A6
